docs/src/modules/components/Demo.js

```diff
@@ -250,16 +269,20 @@ export default function Demo(props) {
         </NoSsr>
       )}
       <Collapse in={openDemoSource} unmountOnExit>
-        <Code
-          id={demoSourceId}
-          code={showPreview && !codeOpen ? demo.jsxPreview : demoData.raw}
-          language={demoData.sourceLanguage}
-          copyButtonProps={{
-            'data-ga-event-category': codeOpen ? 'demo-expand' : 'demo',
-            'data-ga-event-label': demoOptions.demo,
-            'data-ga-event-action': 'copy-click',
-          }}
-        />
+        <div>
+          <CodeEditor
+            key={demoKey}
+            id={demoSourceId}
+            value={code}
+            onChange={setCode}
+            language={demoData.sourceLanguage}
+          />
+          {debouncedError && (
+            <Typography color="error" component="pre" sx={{ whiteSpace: 'pre-wrap' }}>
+              {debouncedError}
+            </Typography>
+          )}
```

Comment:
<img width="793" alt="Screen Shot 2565-05-04 at 16 09 29" src="https://user-images.githubusercontent.com/18292247/166652946-68bdbc18-eca2-47ff-8605-94d6e29316d7.png">

Let's wait for @danilo-leal feedback on the error appearance.

Reply:
<img width="518" alt="image" src="https://user-images.githubusercontent.com/2595058/166661296-43c275d9-b82e-4ef1-8937-b1432c2909c3.png">
I changed to use `FormHelperText`